presto-docs/src/main/sphinx/admin/function-namespace-managers.rst

```diff
@@ -0,0 +1,88 @@
+===========================
+Function Namespace Managers
+===========================
+
+.. warning::
+
+    This is an experimental feature being actively developed. The way
+    Function Namespace Managers are configured might be changed.
+
+Function namespace managers support storing and retrieving SQL
+functions, allowing Presto engine to perform actions such as creating,
+altering, deleting, and executing functions.
+
+A function namespace is in the format of ``catalog.schema`` (e.g.
+``example.test``). It can be thought of as a schema for storing
+functions. However, it is not a full fledged schema as it does not
+support storing tables and views, but only functions.
+
+Each Presto function, whether built-in or user-defined, resides in
```

Comment:
Introduce the concept of "qualified function name first". `A qualified function name is a .... Each Presto function, whether ......`

Reply:
Should we *Italy* the concepts like *function namespace manager*, *function namespace*, *qualified function name*? What's the convention?

Reply:
I don't see those kinds of formatting be used in presto-docs.